src/app/components/signin/signin.component.ts

```diff
@@ -14,18 +17,22 @@ export class SigninComponent implements OnInit {
   public loading: boolean;
   public signinForm: FormGroup;
   public message: any;
+
   constructor(
     private formBuilder: FormBuilder,
     private api: ApiService,
     private router: Router,
-    private authService: AuthService
+    private authService: AuthService,
+    private store: Store<any>
     ) {}
 
   ngOnInit() {
 
     this.loading = false;
-    localStorage.removeItem('_userToken');
-    this.authService.setLoginStatus(false);
+
+    // localStorage.removeItem('_userToken'); TODO - Remove
+    // this.authService.setLoginStatus(false); TODO - Remove
```

Comment:
Here too, There's a few more TODO - Remove comments, just go ahead and remove them. Let's keep TODO comments to places where we want to add or change functionality. If we can affectively remove something by commenting it out, let's just remove it.